website/src/pages/contributing/contributors/index.tsx

```diff
@@ -30,6 +30,7 @@ const Index = () => {
         <ExternalLink to={contributor.url} label={contributor.name} />
       </h2>
       <p>{contributor.bio}</p>
+      <a href={`#${contributor.id} `}>{contributor.name}</a>
```

Comment:
this is not the intended solution. you need something to link to, not a new link ;)

Reply:
@Idrinth alright could you please provide more information for me to well understand and know how to go about the issue please 🙏

Reply:
You need to create a link target, not a link. See the desired solution in the issue :)

Reply:
Hi @Asin-Junior-Honore I am assuming you gave up on the issue?